app/rest/docker/taskLogs.js

```diff
@@ -0,0 +1,18 @@
+angular.module('portainer.rest')
+.factory('TaskLogs', ['$http', 'DOCKER_ENDPOINT', 'EndpointProvider', function ServiceLogsFactory($http, DOCKER_ENDPOINT, EndpointProvider) {
+  'use strict';
+  return {
+    get: function (id, params, callback) {
+      $http({
+        method: 'GET',
+        url: DOCKER_ENDPOINT + '/' + EndpointProvider.endpointID() + '/tasks/' + id + '/logs',
+        params: {
+          'stdout': params.stdout || 0,
+          'stderr': params.stderr || 0,
+          'timestamps': params.timestamps || 0,
+          'tail': params.tail || 'all'
+        }
+      }).success(callback).error(function (data, status, headers, config) {});
```

Comment:
What is the point of ignoring request failure? I think we need to display a notification if the request is failed.